tests/test_status.py

```diff
@@ -0,0 +1,29 @@
+import pytest
+
+from alfasim_sdk.status import ErrorMessage
+from alfasim_sdk.status import WarningMessage
+
+
+@pytest.mark.parametrize("status", [WarningMessage, ErrorMessage])
+def test_status(status):
+    out = status(model_name="Test1", message="Message From test")
+    assert out.model_name == "Test1"
+    assert out.message == "Message From test"
+
+    with pytest.raises(ValueError, match='The field "model_name" cannot be empty'):
```

Comment:
TBH I don't see much point in this test, you are basically testing `attr.s` functionality...

Reply:
Yep, basically, I'm just testing that the declaration has a validator and no one has removed/overwritten by mistake or something similar.